compiler/quilt/test/test_core.py

```diff
@@ -0,0 +1,37 @@
+"""
```

Comment:
This doesn't seem like the right way to enforce that the registry and compiler are using the same core.py. How about a commit hook? Or, better yet, having a single file and import it into both.